projects/igniteui-angular/src/lib/expansion-panel/expansion-panel-header.component.html

```diff
@@ -0,0 +1,16 @@
+
+<div tabindex="0" role = "button" [attr.aria-disabled] = "disabled" [attr.aria-expanded] = "isExpanded" [attr.aria-controls] = "controls">
+<ng-template #iconContainer>
+    <ng-content *ngIf="iconTemplate" select="igx-expansion-panel-icon"></ng-content>
+    <igx-icon *ngIf="!iconTemplate" fontSet="material" [name]="panel.collapsed? 'expand_more':'expand_less'"></igx-icon>
+</ng-template>
+<ng-container *ngIf="iconPosition === 'left'">
+    <ng-container *ngTemplateOutlet="iconContainer"></ng-container>
+</ng-container>
+<ng-content select="igx-expansion-panel-title"></ng-content>
+<ng-content select="igx-expansion-panel-description"></ng-content>
```

Comment:
Can you also add CSS classes for the header title and description? For instance `igx-expansion-panel__header-title` and `igx-expansion-panel__header-description`.